### PR TITLE
Implement "doctor" command

### PR DESCRIPTION
**Pull request by teobugslayer**
See https://github.com/NativeScript/nativescript-cli/issues/480

### CODE REVIEW

File: commands/doctor.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		return future.fromResult(true);
+	}
+
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			var warningsPrinted = this.$doctorService.printWarnings();
+			if (warningsPrinted) {
+				this.$logger.out("Please note that these warnings are just used to help the NativeScript maintainers".bold + os.EOL +
```
Comment (rosen-vladimirov) [inline commands/doctor.ts]: NativeScript -> `${this.$staticConfig.CLIENT_NAME_ALIAS || this.$staticConfig.CLIENT_NAME}`

Comment (rosen-vladimirov) [inline commands/doctor.ts]: and same is valid for other NativeScript occurrences

**Comment by teobugslayer** [comment]
I will make a separate PR for the AppBuilder to restore the missing functionality

File: commands/doctor.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+	public canExecute(args: string[]): IFuture<boolean> {
+		return future.fromResult(true);
+	}
+
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			var warningsPrinted = this.$doctorService.printWarnings();
```
Comment (rosen-vladimirov) [inline commands/doctor.ts]: var -> let

File: commands/doctor.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+			if (warningsPrinted) {
+				this.$logger.out(`These warnings are just used to help the ${this.$staticConfig.CLIENT_NAME_ALIAS || this.$staticConfig.CLIENT_NAME} maintainers with debugging if you file an issue.`.bold + os.EOL +
```
Comment (rosen-vladimirov) [inline commands/doctor.ts]: you can get the clientName in a separate variable

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**